CMakeLists.txt

```diff
@@ -500,7 +500,7 @@ elseif(OPENGL_FOUND)
         ${OPENGL_INCLUDE_DIR}
```

Comment:
When you use OpenGL::GL, the include dir is probably included ~ this might not be needed any more.

Reply:
... unless it refers back to e.g. GLEW or the selected library ....

Reply:
> When you use OpenGL::GL, the include dir is probably included ~ this might not be needed any more.

I agree with @meshula  observations, see

https://github.com/Kitware/CMake/blob/master/Modules/FindOpenGL.cmake#L684

Reply:
Agree, good catch! Thanks!